roadblock.py

```diff
@@ -976,16 +972,25 @@ def message_publish(self, message):
                 break
 
             counter += 1
-            # this call should return the number of clients that receive the message
-            # we expect it to be greater than zero, if not we retry
-            ret_val = self.redcon.publish(self.roadblock_uuid + "__busB", message_str)
 
-            if ret_val == 0:
+            try:
+                # need to figure out the foo:bar thing...it feels like
+                # we don't really need information there but it
+                # appears to be a required field :/
```

Comment:
perhaps you are creating the bus without the required fields so it does that by default?

Reply:
This was one thing I wanted to resolve and I had forgotten about it.  I just figured this out and removed it.